packages/tailwindcss-language-server/tests/completions/completions.test.js

```diff
@@ -281,7 +281,7 @@ withFixture('basic', (c) => {
 
     expect(resolved).toEqual({
       ...item,
-      detail: '--tw-bg-opacity: 1; background-color: rgb(239 68 68 / var(--tw-bg-opacity));',
+      detail: '--tw-bg-opacity: 1; background-color: rgb(239 68 68 / var(--tw-bg-opacity, 1));',
```

Comment:
Is this change from the v3 upgrade? 🤔

Reply:
Yep, we had an old 3.4 version in the fixtures.